505.xml

```diff
@@ -0,0 +1,265 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- 
+FILE INFORMATION
+OMA Permanent Document
+   File: OMA-SUP-XML_505-V1_0-YYYY-MM-DD-A
+   Path: https://www.openmobilealliance.org/release/ObjLwM2M_Remote_SIM_Provisioning
+Public Reachable Information
+   Path: https://github.com/OpenMobileAlliance/lwm2m-registry
+   Name: 505.xml
+NORMATIVE INFORMATION
+  Send comments to https://github.com/OpenMobileAlliance/OMA_LwM2M_for_Developers/issues
+LEGAL DISCLAIMER
+Copyright 2021 Open Mobile Alliance. 
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions
+are met:
+1. Redistributions of source code must retain the above copyright
+notice, this list of conditions and the following disclaimer.
+2. Redistributions in binary form must reproduce the above copyright
+notice, this list of conditions and the following disclaimer in the
+documentation and/or other materials provided with the distribution.
+3. Neither the name of the copyright holder nor the names of its
+contributors may be used to endorse or promote products derived
+from this software without specific prior written permission.
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+The above license is used as a license under copyright only.  Please
+reference the OMA IPR Policy for patent licensing terms:
+https://www.omaspecworks.org/about/intellectual-property-rights/
+-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://openmobilealliance.org/tech/profiles/LWM2M-v1_1.xsd">
+<Object ObjectType="MODefinition">
+		<Name>LwM2M Remote SIM Provisioning</Name>
```

Comment:
The term "Remote SIM Provisioning" includes both SIM profile download and SIM profile management. Clearly, in both object 504 and 505 LwM2M is used to initiate profile download and for profile management. For this reason, by using "LwM2M" in the object name will be misleading for the user that is searching for a Remote SIM Provisioning object to use with LwM2M. Therefore, we cannot accept usage of "LwM2M" in the object name. Our proposal is "nuSIM Provisioning" because this is the only Remote SIM Provisioning solution known to us that fits the proposed set of resources.

Reply:
OK, we are trying a new one: "LwM2M nuSIM Provisioning".  If that works for you, please mark this conversation as resolved,'

Reply:
We are OK with the object name "LwM2M nuSIM Provisioning".